WebDevStudios/wds-bt
# fix / unsafe svg

# Closes

#42 

## Link to test

coming soon...

## Description

Removes `enable-svg.php` - there was no sanitization of svgs.

If we need SVG support we can use a safe SVG plugin instead.

## Related Tickets & Documents

N/A

## Mobile & Desktop Screenshots/Recordings

N/A

## Added to documentation?

- [x] 🙅 No documentation needed (Removing a feature that was not documented)

## Added tests?


## Testing Instructions

1. Pull this branch
2. Verify that SVG support has been removed from the theme

-----

## Reviewer's Testing Checklist



As a reviewer, please verify that the relevant testing criteria are fulfilled and confirmed before approving this Pull Request.


## [optional] Additional Reviewer Notes or Considerations?
